### PR TITLE
Measure the processing time of each smoke test and record the result in debug_messages

### PR DESCRIPTION
**Pull request by ykitamura-mdsol**
Record the processing time in `debug_messages`

It will show up like this on rake task:
screen shot 2016-11-04 at 4 05 30 pm

@jfeltesse-mdsol @jcarres-mdsol @cabbott @ssteeg-mdsol

### CODE REVIEW

**Comment by JordiPolo** [comment]
I think this is worth a bigger update. 
`duration` or `processing_time` , (we need a native) can be a method in `Result` also in `EvaluationCollection` then they can be top level elements in the json output, console output looks good

**Comment by JordiPolo** [comment]
Also would need to update docs. This time is in `ms` ?

**Comment by ykitamura-mdsol** [comment]
The time is in `second` now, but it might be good to convert in `ms`

**Comment by jfeltesse-mdsol** [comment]
LGTM

**Comment by ykitamura-mdsol** [comment]
addressed comments

added `ms` on rake output
screen shot 2016-11-07 at 5 37 23 pm

**Comment by ssteeg-mdsol** [comment]
Change looks good - `duration` works well. 👍

File: lib/kapnismology/result.rb [hunk 1]
```diff
@@ -38,6 +42,10 @@ def format_debug_messages(debug_messages)
     def format_passed(passed)
       passed ? Terminal.green('passed') : Terminal.red('failed')
     end
+
+    def format_duration(duration)
+      duration ? "duration: #{Terminal.bold(duration)} ms\n" : ''
```
Comment (jcarres-mdsol) [inline lib/kapnismology/result.rb]: is possible for duration to not be set?

Comment (jcarres-mdsol) [inline lib/kapnismology/result.rb]: Maybe should be initialized to 0 in all the result classes

**Comment by ykitamura-mdsol** [comment]
addressed comments

**Pull request merged**